docs-v2/guides/api-unification.mdx

```diff
@@ -67,54 +67,65 @@ To build a custom unified API in Nango, follow these steps:
 
 ### 1. Define your unified model
 
-Start by defining your unified model in the `nango.yaml` file. This is where you standardize the fields and structure for your use case:
-
-```yaml
-models: 
-  User:
-    id: string
-    firstName: string
-    lastName: string
-    email: string
+Start by defining your unified model with a zod model in a file, e.g: `models.ts`.
+
+```ts models.ts
+export const UserUnified = z.object({
+  id: z.string(),
+  name: z.string(),
+  email: z.string().email(),
+});
 ```
 
 ### 2. Define your unified endpoints
 
-In your `nango.yaml` configuration, standardize the configuration of endpoints across APIs for consistency: use the same method, path & parameters across APIs.
-
-```yaml
-integrations:
-  jira-basic:
-    actions:
-      create-user:
-        description: Creates a user in Jira.
-        output: User # Common output between Jira & Zendesk
-        endpoint:  # Common endpoint between Jira & Zendesk
-          method: POST
-          path: /users
-          group: Users
-        input: User  # Common input between Jira & Zendesk
-  zendesk:
-    actions:
-      create-user:
-        description: Creates a user in Zendesk.
-        output: User
-        endpoint:
-          method: POST
-          path: /users
-          group: Users
-        input: User
-...
+In your script's configuration, standardize the configuration of endpoints across APIs for consistency: use the same method, path & parameters across APIs.
+
+<CodeGroup>
+```ts jira/create-user.ts
+import { createAction } from 'nango';
+import { UserUnified } from './models.js';
+
+export default createAction({
+  endpoint: {
+    method: 'POST',
+    path: '/users',
+    group: 'Users',
+  },
+  input: UserUnified,
+  output: UserUnified,
+  exec: async (nango, input) => {
+    // Fetch from Jira
```

Comment:
Is "fetch" the correct term here? I would have thought this was creating the user on Jira

Reply:
true